Fix blood and stamina constantly flashing on regain tick

Mehuge opened the pull request.

codecorsair commented on game/hud/src/components/PlayerStatusComponent/index.tsx:
```diff
@@ -169,39 +171,43 @@ class PlayerStatusComponent extends React.Component<PlayerStatusComponentProps,
                  valuePerPill={VALUE_PER_BODY_PARTY_PILL}
                  currentValue={this.props.playerStatus.health[BodyParts.RightArm].current}
                  maxValue={this.props.playerStatus.health[BodyParts.RightArm].maximum}
+                 flashThreshold={0}
```
Body parts should have a flash threshold set as well as they have regen, you should be able to see that regen working by doing /ouch in game since abilities are disabled currently.

Mehuge commented:
Updated to set flash threshold on body parts.  It is currently doing 0.5% on my character so set threshold to 1%.  Similarly set blood and stamina thresholds to 1%.  Moved threshold values into consts at top of module for easy tweaking.
